src/main/java/frc/robot/commands/CDSReverseCommand.java

```diff
@@ -11,21 +11,21 @@
 public class CDSReverseCommand extends CommandBase {
   /** Creates a new IntakeForwardCommand. */
   private final CDSSubsystem mCDSSubsystem;
-  private final ShooterSubsystem mShooterSubsystem;
+  //private final ShooterSubsystem mShooterSubsystem;
```

Comment:
Why did we remove Shooter subsystem from the CDS commands? How do they coordinate the belt -> cargo stopper action?